### PR TITLE
Extending ns-render to support loop rendering using interpolated cameras

### PR DESCRIPTION
**Pull request by ponimatkin**
This PR extends possibility of ns-render utility to render the data using already available camera poses. It is helpful when working with large datasets (ScanNet, ARKitScenes) as well as HPC clusters where computational nodes are not connected to the internet.

### CODE REVIEW

File: scripts/render.py [hunk 1]
```diff
@@ -255,7 +259,7 @@ class RenderTrajectory:
     """Path to config YAML file."""
     rendered_output_names: List[str] = field(default_factory=lambda: ["rgb"])
     """Name of the renderer outputs to use. rgb, depth, etc. concatenates them along y axis"""
-    traj: Literal["spiral", "filename"] = "spiral"
```
Comment (tancik) [inline scripts/render.py:258]: Now that we are adding more options, can you add a description of each option in the class docstring.

**Review by tancik — COMMENTED** [review]
Thanks for adding! As you probably guessed, we had this at some point a long time ago, but it never made it into the render.py script.

File: scripts/render.py (hunk 1 above)
Comment (ponimatkin) [reply scripts/render.py:258]: I tried expanding docstring a bit in https://github.com/nerfstudio-project/nerfstudio/pull/1508/commits/cf01d605a1b67f6ea435d8cac37ff8a07f9376c9, do you think it is good or a bit verbose?

Comment (tancik) [reply scripts/render.py:258]: Sorry, I should have been more specific. I was thinking the main docstring could be updated to something like,

```
"""Load a checkpoint, render a trajectory, and save to a video file.
The following trajectory options are available,
filename: Load from trajectory created using viewer or blender vfx plugin.
interpolate: Create trajectory by interpolating between eval dataset images.
spiral: Create a spiral trajectory (can be hit or miss).
"""
```

File: scripts/render.py [hunk 2]
```diff
@@ -255,8 +258,9 @@ class RenderTrajectory:
...
-    """Trajectory to render."""
+    traj: Literal["spiral", "filename", "loop"] = "spiral"
```
Comment (tancik) [inline scripts/render.py:261]: I was thinking a little more about the naming. Perhaps another term like `interpolate` would be more descriptive than `loop`

Comment (ponimatkin) [reply scripts/render.py]: Ah I see! Sorry for the confusion. :)

In this commit I've added this here https://github.com/ponimatkin/nerfstudio/commit/b8f7eb8a17ea35abec1c422db109d1eb3f088e2e. Then here https://github.com/ponimatkin/nerfstudio/commit/1df0ea87ab9fee0bd5570174e01ecd1a6b0e4c61 I've actually went further on and removed any mentions of the loops.

File: scripts/render.py (hunk 1 above)
Comment (ponimatkin) [reply scripts/render.py:258]: Ah I see! Sorry for the confusion. :)

I've added your suggestions here b8f7eb8a17ea35abec1c422db109d1eb3f088e2e.

**Review by tancik — APPROVED** [review]
LGTM, thanks for adding this functionality.

**Pull request merged**